types/d3/v3/d3-tests.ts

```diff
@@ -786,7 +786,7 @@ function populationPyramid() {
         .scale(y)
         .orient("right")
         .tickSize(-width)
-        .tickFormat(function (d) { return Math.round(d / 1e6) + "M"; } );
+        .tickFormat(function (d,i) { console.log(i); return Math.round(d / 1e6) + "M"; } );
```

Comment:
Try to make the test more "precise": `console.log` works whatever the type of `i` is.
Also a tests for the getter and with one parameter, to be sure it still works, would be nice.

Please respect the file style indentation:
```suggestion
        .tickFormat(function (d, i) { console.log(i); return Math.round(d / 1e6) + "M"; } );
```
Same below: `(t: any, index: number)`